buildspec.yml

```diff
@@ -18,10 +18,10 @@ phases:
       - echo Build started on `date`
       - make -C src minisat2-download
       - make -C jbmc/src setup-submodules
-      - make -C src CXX="ccache g++" -j2
-      - make -C unit CXX="ccache g++" -j2
-      - make -C jbmc/src CXX="ccache g++" -j2
-      - make -C jbmc/unit CXX="ccache g++" -j2
+      - make -C src CXX="ccache g++" CXXFLAGS="-D_GLIBCXX_DEBUG" -j2
+      - make -C unit CXX="ccache g++" CXXFLAGS="-D_GLIBCXX_DEBUG" -j2
+      - make -C jbmc/src CXX="ccache g++" CXXFLAGS="-D_GLIBCXX_DEBUG" -j2
+      - make -C jbmc/unit CXX="ccache g++" CXXFLAGS="-D_GLIBCXX_DEBUG" -j2
```

Comment:
Note that you are completely replacing `CXXFLAGS` at this point - you might actually want `CXXFLAGS="-Wall -pedantic -Werror -Wno-deprecated-declarations -Wswitch-enum -D_GLIBCXX_DEBUG"`